lib/browser.js

```diff
@@ -198,19 +198,25 @@ PaymentProtocol.verifyCertChain = function(chain, sigHashAlg) {
     if (sigHashAlg !== 'none') {
       var sigAlgId = c.signatureAlgorithm.algorithm.join('.')
       var sigAlg = PaymentProtocol.getAlgorithm(sigAlgId).split('_')
-      var jsrsaSig = new KJUR.crypto.Signature({
-        alg: sigAlg[1].toLowerCase() + 'withRSA',
-        prov: 'cryptojs/jsrsa'
-      });
-      jsrsaSig.init(npubKey);
-
-      // Get the raw DER TBSCertificate
-      // from the DER Certificate:
-      var tbs = PaymentProtocol.getTBSCertificate(data, sig);
-
-      jsrsaSig.updateHex(tbs.toString('hex'));
-
-      sigVerified = jsrsaSig.verify(sig.toString('hex'));
+      // If the signature algorithm is plain RSA, the cert is invalid
+      console.log(sigAlg)
```

Comment:
Please, remove your `console.log`.

Reply:
That was already removed and committed days ago
https://github.com/Bitcoin-com/bitcore-payment-protocol/pull/1/commits/d37211de345f74b2b0b24b43eebae990ce1015ee